docs/dev_guides/core_concepts.md

```diff
@@ -0,0 +1,55 @@
+# Core concepts
+
+`napari-threedee` provides reusable components (called **`threedee` objects**) which 
+- enable custom 3D interactive functionality
+- simplify the development of workflows with 3D interactivity
+
+
+
+## How to use `napari-threedee`
+
+`napari-threedee` is designed to be used both as a **napari plugin** and as a **library**.
+
+When used as a **plugin**, a set of widgets provide custom 3D manipulation and 
+annotation tools in napari. In this case, users need not delve into the code.
+
+For those who want to integrate custom 3D interactivity into their application, 
+`napari-threedee` can also be used as a **library**. In this case, users can
+- integrate existing functionality into their applications
+- develop new custom functionality more easily
+
+## `threedee` objects
+**`threedee` objects** can be split into two categories, **manipulators** and **annotators**.
+They can be activated and deactivated as required.
+
+### Manipulators
+A **Manipulator** is an object in the scene which allows us to manipulate something. 
```

Comment:
```suggestion
A **Manipulator** is an object in the scene which allows us to position something. 
```

just to avoid using the word in the definition.